doc/source/examples/extended_examples/executable/executable.rst

```diff
@@ -101,6 +101,64 @@ This code shows how you can input other arguments:
    The first natural frequency is 344.28 Hz.
 
 
+Convert the app to an executable
+================================
+
+Using the Python library `PyInstaller <https://pyinstaller.org>`, it is possible to convert the app to an executable by taking a few precautions.
+
+.. code:: console
+
+   pyi-makespec cli_rotor.py
+
+Add the argument ``--onefile`` to put everything in a single file. Otherwise you will have a folder containing the application and dll dependencies.
+
+.. code:: console
+
+   pyi-makespec cli_rotor.py --onefile
+
+After the first line in the file ``cli_rotor.spec``, add the following lines:
```

Comment:
Why? I guess you want to add those files to the executable? Please explain.

Reply:
Also, why those files? 

I don't think `os.path.join(proot, "VERSION")` exists.......

Reply:
> Also, why those files?
> 
> I don't think `os.path.join(proot, "VERSION")` exists.......

Why it wouldn't exist ?

Reply:
if `proot` is a the root of the repo (or at `src/ansys/mapdl/core`) there is not such a file called `VERSION`. Or am I missing something?

Reply:
the folder `ansys.api.mapdl` (which is where proot points to) correspond to the library [ansys-api-mapdl](https://github.com/ansys/ansys-api-mapdl)